feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go

```diff
@@ -241,7 +239,7 @@ func TestFabricReboot(t *testing.T) {
 		}
 	}
 	if removableFabric == "" {
-		t.Fatalf("Component(fabric).Removable().Get(t): got none, want non-empty")
+		t.Skipf("No fabric component found for the testing")
```

Comment:
Same as above.

Reply:
Thanks for your comments. Updated script as below:
```go
		if *args.NumLinecards > 0 {
			t.Fatalf("No removable fabric component found for the testing on a modular device")
		} else {
			t.Skipf("No removable fabric component found for the testing")
		}
```